runhouse/servers/http/http_server.py

```diff
@@ -659,9 +575,10 @@ async def call(
     ):
         kwargs = args.get("kwargs", {})
         args = args.get("args", [])
+        token = get_token_from_request(request)
```

Comment:
We should hash the token here so the only thing that's passed around / through the servlet is the hash, not the token itself. The AuthCache's lookup should accept the hash, not the token.

Reply:
Hmm any ideas then on how we can refresh the cache? Bc we need to pass a token into the request to den